pkg/collector/prometheus_collector_test.go

```diff
@@ -127,7 +130,10 @@ var _ = Describe("Test Prometheus Collector Unit", func() {
 		exporter.NodeMetrics.SetNodeComponentsEnergy(componentsEnergies)
 		exporter.NodeMetrics.UpdateIdleEnergy()
 		exporter.NodeMetrics.UpdateDynEnergy()
-		model.UpdateContainerEnergyByRatioPowerModel(*exporter.ContainersMetrics, exporter.NodeMetrics)
+		var wg sync.WaitGroup
+		wg.Add(1)
```

Comment:
add 1 and wait? what's benefit as we move to go routine?

Reply:
Because in the test we are only updating the energy of only one source.
But the code updates more.